core/terraform/aws/variables.tf

```diff
@@ -31,13 +31,13 @@ variable "ami" {
   description = "Amazon Machine Image"
 }
 
-variable "access_key" {
-  description = "AWS Access Key"
-}
+#variable "access_key" {
+#  description = "AWS Access Key"
+#}
 
-variable "secret_key" {
```

Comment:
how is this not needed anymore? I have seen it in the props still. I think we need to keep it? @moliele

Reply:
If we use profile, these are not used automatically. So this is a case dependant on the previous comment.  https://github.com/confluentinc/confluent-hybrid-cloud-workshop/pull/63/files#r1070994255